agent/src/main/java/org/openremote/agent/protocol/http/AbstractHTTPServerProtocol.java

```diff
@@ -38,6 +38,7 @@
 import org.openremote.model.asset.agent.Agent;
 import org.openremote.model.asset.agent.AgentLink;
 import org.openremote.model.attribute.Attribute;
+import org.openremote.model.http.HTTPMethod;
```

Comment:
?

Reply:
Result of moving HTTPMethod enum from Agent to Model module.